js/store.js

```diff
@@ -360,6 +363,29 @@ const store = {
     // empty out request processing queue
     capture.queue = [];
     return await this.db.websites.clear();
+  },
+
+  async getOldestDate() {
+    const oldestSite = await this.db.websites.orderBy('requestTime').first();
+    if (!oldestSite) {
+      return '';
```

Comment:
I think this should return false, this blank string is actually presentation. Looks like you already would cope with that in lightbeam anyway.